RecipeBook/ViewControllers/HomeScreenViewController/HomeScreenViewController.swift

```diff
@@ -27,6 +32,7 @@ class HomeScreenViewController: UIViewController {
         setupUI()
         setupCategoriesCollectionView()
         loadAllCategories()
+        setupRecommendationsCollectionView()
```

Comment:
Just for clarity, leave the setup methods first and the methods for loading the data last.